src/main/java/me/moodcat/database/entities/Room.java

```diff
@@ -45,28 +51,53 @@
     private Integer id;
 
     /**
+     * The name of the room.
+     */
+    @Column(name = "roomName")
+    private String roomName;
```

Comment:
But response models should be different than databate entities and instances anyway.